nautobot/extras/models/change_logging.py

```diff
@@ -133,6 +133,22 @@ class Meta:
                 name="extras_objectchange_rtime_idx",
                 fields=["-time"],
             ),
+            models.Index(
+                name="changed_object_double_idx",
+                fields=["changed_object_type", "changed_object_id"],
+            ),
+            models.Index(
+                name="related_object_double_idx",
+                fields=["related_object_type", "related_object_id"],
+            ),
+            models.Index(
+                name="user_changed_object_triple_idx",
+                fields=["user", "changed_object_type", "changed_object_id"],
+            ),
+            models.Index(
+                name="name_changed_object_triple_idx",
+                fields=["user_name", "changed_object_type", "changed_object_id"],
+            ),
```

Comment:
Do we need/want a combo index on `user/related_object` and `user_name/related_object` as well? I know it wasn't called out in #5012 but it "feels" like a gap to me. @lampwins any input?

Reply:
Maybe. The related objects are very rarely used and thus even less accessed. I'd be fine leaving them out for now, unless you guys really want to add them.